scripter/scripter.go

```diff
@@ -154,3 +159,24 @@ func ReloadAllScripters(scripters map[string]Scripter) {
 		ReloadScripts(script)
 	}
 }
+
+// SetConnectionChecker sets an interval of 5 minutes that checks all connections on their last use.
+// When not used for over an hour, the connection is removed from the list and therefor collected by the garbage
+// collector, clearing unused memory space.
+func SetConnectionChecker(scripters map[string]Scripter) {
+	ticker := time.NewTicker(cleanupInterval * time.Second)
+	connectionChecker := make(chan struct{})
```

Comment:
Since the channel is in local scope, wouldn't calling it `stop` or `quit` be more logical?